src/functionalTest/java/uk/gov/hmcts/reform/blobrouter/config/TestConfiguration.java

```diff
@@ -11,6 +11,7 @@
     public final String sourceStorageAccountKey;
     public final String sourceStorageAccountUrl;
     public final String crimeSourceContainer;
+    public final Boolean useProxyForSourceStorage;
```

Comment:
why `Boolean` and not `boolean`?

Reply:
I want it to be null as well.

Reply:
What's the reason for that?

Also in constructor `boolean getBoolean(String path);` is called to set this field, so I think it is never null.

Reply:
to avoid setting TEST_USE_PROXY_FOR_SOURCE_STORAGE in cnp jenkins. as I said in my comment conf does not support null.